From 947643509279a605a09959a06d332bf027e8be57 Mon Sep 17 00:00:00 2001
From: Mikhail Kshevetskiy <mikhail.kshevetskiy@iopsys.eu>
Date: Mon, 10 Nov 2025 06:56:43 +0300
Subject: [PATCH] dt-bindings: clock: airoha: update EN7523 binding for reset support

Update EN7523 clock binding YAML: remove the '#reset-cells: false'
restriction that was preventing EN7523 from using the reset controller
support added by the accompanying driver patch, and add '#reset-cells: <1>'
to the EN7523 compatible example.

Signed-off-by: Mikhail Kshevetskiy <mikhail.kshevetskiy@iopsys.eu>
Reviewed-by: Rob Herring (Arm) <robh@kernel.org>
Signed-off-by: Stephen Boyd <sboyd@kernel.org>
---
 Documentation/devicetree/bindings/clock/airoha,en7523-scu.yaml | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/Documentation/devicetree/bindings/clock/airoha,en7523-scu.yaml
+++ b/Documentation/devicetree/bindings/clock/airoha,en7523-scu.yaml
@@ -64,8 +64,6 @@ allOf:
             - description: scu base address
             - description: misc scu base address
 
-        '#reset-cells': false
-
   - if:
       properties:
         compatible:
@@ -89,6 +87,7 @@ examples:
       reg = <0x1fa20000 0x400>,
             <0x1fb00000 0x1000>;
       #clock-cells = <1>;
+      #reset-cells = <1>;
     };
 
   - |
